srcs/incs/CommonDefinitions.hpp

```diff
@@ -0,0 +1,9 @@
+#ifndef SRCS_INCS_COMMONDEFINITIONS_HPP_
+#define SRCS_INCS_COMMONDEFINITIONS_HPP_
+
+#include <arpa/inet.h>
+
+extern const char valid_http_methods[8][8];
+extern const uint16_t valid_return_status[5];
```

Comment:
What do you think about creating a namespace for the constants variables and defining them here?

Example:
```
namespace constants {
const char VALID_HTTP_METHODS[8][8] = {"GET", "HEAD", "POST",
	"PUT", "DELETE", "CONNECT", "OPTIONS", "TRACE"};
const uint16_t VALID_RETURN_STATUS[] = {301, 302, 303, 307, 308};
// Other constants ...
}
```

Reply:
Look [here](https://stackoverflow.com/questions/2190919/mixing-extern-and-const). Inline constexpr would be the preferred way to do in in C++, but it's C++17. We have constexpr, but they are C++11.